### PR TITLE
[AKS] az aks create: add --enable-aad argument

### PR DESCRIPTION
**Pull request by bingosummer**
**Description**  

This PR is to portal the support of creating AKS-managed Azure AD cluster from aks-preview extension. It adds a new argument `--enable-aad` to `az aks create` command. Customers can specify the `--aad-admin-group-object-ids` and `--aad-tenant-id` when creating/updating AKS cluster.

**Testing Guide**  

* Create an AKS-managed Azure AD cluster

```
az aks create -g MyResourceGroup -n MyManagedCluster --enable-aad [--aad-admin-group-object-ids <id>] [--aad-tenant-id <id>]
```

* Update an AKS-managed Azure AD cluster

```
az aks update -g MyResourceGroup -n MyManagedCluster [--aad-admin-group-object-ids <id>] [--aad-tenant-id <id>]
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -222,6 +222,12 @@
   - name: --windows-admin-password
     type: string
     short-summary: Password to create on Windows node VMs.
+  - name: --enable-aad
+    type: bool
+    short-summary: Enable managed AAD feature for cluster.
+  - name: --aad-admin-group-object-ids
+    type: string
+    short-summary: Comma seperated list of aad group object IDs that will be set as cluster admin.
```
Comment (arrownj) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:230]: Usually we use space separated for this case. And add `nargs='+'` for this argument in `_params.py`. There are so many examples in our code base, take [this ](https://github.com/Azure/azure-cli/blob/b5937aa49d572fe93e02bb2d30fa6b1cc575d224/src/azure-cli/azure/cli/command_modules/acr/_params.py#L144)for example.

If there is no special requirement for this parameter, I strongly suggest to follow above pattern. With that, we do not need to handle the parse logic ourselves.

Comment (bingosummer) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py:230]: Thanks for reviewing the PR. In azure-cli-extension, we use the comma separated list for this argument. Is it OK that we use space separated for azure-cli and comma separated for azure-cli-extension?

Comment (arrownj) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py:230]: If they are the same parameter, it's better to keep them the same.

**Pull request merged**